src/components/redemption/Start.js

```diff
@@ -21,7 +22,14 @@ const Start = ({ saveAddresses, resetState, openWalletModal }) => {
     isValid: false,
     hasError: false,
   }
-  const [depositAddress, setDepositAddress] = useState(initialAddressState)
+
+  const location = useLocation()
+  const searchParams = new URLSearchParams(location.search)
+
+  const [depositAddress, setDepositAddress] = useState({
+    ...initialAddressState,
+    address: searchParams.get("deposit") || "",
+  })
```

Comment:
Thoughts on routing `/deposit/:address/redeem` to this page? That will match our other routes and set us up for some of the stuff in the new design while maintaining route stability, which is desirable.

Reply:
I like it - I approached it like this initially, wrapped in Loadable, but ended up with an infinite loop. I'll have another go.